packages/api/helper/nominationGsheetToDB.js

```diff
@@ -76,8 +78,13 @@ module.exports = function gsheetToDB() {
           //db.Nomination.findOrCreate returns a promise that when fulfilled returns an array with two elements 
           //the first element is the nomination instance object that contains the dataValues among other things  
           //the second element is a boolean that represents whether a nomination was created
-          if (array[1]) {
-            verifyHcEmail(array[0].dataValues)
+          if(array[1]) {
+            let hasProviderBeenValidated = dbNominations.some((dbNom) => {
```

Comment:
Changes made, John will apply the formatting changes.

Reply:
Hey Austin, just wanted to point this out before it gets tested.  So currently the googlesheet containing the nominations is shared by multiple environments (locally, staging, production).  When a single nomination is added to the sheet, and the button 'sync nominations' is clicked, depending on whatever application is running (localhost:3000, staging website, production website) it will create a new nomination within the database for that environment.  Since there's 3 env that I know of technically 3 nominations will be created one on each database (assuming the 'sync nominations' button is clicked while running the applications in all env).    So if the provider email address has been validated in one database and not the other, then a verification link email might still be sent.  This is possible because the verification link address depends on the env variable which varies depending on what env you were running the application and creating the nomination (via the 'sync nomincations' button).  I spoke to Bryan about this to some extent.